pkg/server/service.go

```diff
@@ -228,6 +230,12 @@ func (c *criContainerdService) Run() error {
 		close(grpcServerCloseCh)
 	}()
 
+	// Start profiling server if enable.
+	if c.config.EnableProfiling {
+		glog.V(2).Info("Start profiling server")
+		go startProfilingServer(c.config.ProfilingAddress, c.config.ProfilingPort)
```

Comment:
I feel like that we should move this to `main`. This is not business logic related, let's move it out of `pkg/server`. :)

Reply:
Thanks for your review. will do as your suggestion..

Reply:
Done